kafka/integration_test.go

```diff
@@ -828,21 +828,135 @@ func (its *IntegrationTestSuite) TestAdminClient_DeleteConsumerGroups() {
 		t.Errorf("Consumer group %s should not be present\n", groupID)
 		return
 	}
+	return
 }
 
 // TestAdminClient_ListAndDescribeConsumerGroups validates the working of the
 // list consumer groups and describe consumer group APIs of the admin client.
 //
-// We test the following situations:
+// We test the following situations[Classic Protocol]:
```

Comment:
Question: shouldn't we test these three situations both with classic and consumer protocol?
Listing a group with one client, two client, and an empty consumer group all seem to be legitimate operations even with the Consumer Protocol?

Reply:
Those have already been taken care by from the previous tests, we are just adding an OPTION for request and response to convey for the group type. I do not see any need of having different clients.